app/src/pages/Group/Group.jsx

```diff
@@ -6,6 +6,7 @@ import { useUrlContext } from 'hooks';
 import { Loading, Tabs } from 'components';
 import { groupActions, groupSelectors } from 'redux/groups';
 import { competitionActions } from 'redux/competitions';
+import saveCsv from 'save-csv';
```

Comment:
Can you move this import to above all the local imports? under "react-helmet"

Reply:
Movie groovied